### DISCUSSION TITLE
Lesson 4 - 3:44:00

### DISCUSSION THREAD

**Discussion by shucks18** [smartcontractkit/full-blockchain-solidity-course-py#905] [Q&A] [answered]
im supposed to get a bunch of stuff in the terminal but I get this
![image](https://user-images.githubusercontent.com/81901871/151740216-3a333445-30b2-4789-85cf-41b12b048a63.png)

**Comment by shucks18** [comment]
THIS IS MY CODE
from solcx import compile_standard, install_solc

with open("./SimpleStorage.sol", "r") as file:
    simple_storage_file = file.read()
    print(simple_storage_file)

    install_solc("0.6.0")

    compiled_sol = compile_standard(
        {
            "language": "solidity",
            "sources": {"SimpleStorage.sol": {"content": simple_storage_file}},
            "settings": {
                "outputSelection": {
                    "*": {"*": ["abi", "metadata", "evm.bytecode", "evm.sourceMap"]}
                }
            },
        },
        solc_version="0.6.0",
    )
    print(compiled_sol)

**Comment by shucks18** [comment, marked as answer]
Issue resolved "language": "**Solidity**"

**Reply by cromewar** [reply]
Nice One!
